css/slashed-core.css

```
@@ -0,0 +1,1419 @@
+  .container--full   { max-width: var(--container-full); }
+  .container--prose  { max-width: var(--container-prose); }
+
+  /* ----- Content grid (breakout pattern) ----- */
+  /* ⚠️ --breakout-width must be >= --content-width.
+     If content-width > breakout-width, the calc() yields a negative value
+     and the breakout columns collapse to 0. */
+
+  .content-grid {
+    --content-width: var(--container-default);
+    --breakout-width: var(--container-wide);
+    display: grid;
```

Stylelint (17.9.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)